src/main/java/com/yanolja_final/domain/user/exception/EmailSendingException.java

```diff
@@ -0,0 +1,15 @@
+package com.yanolja_final.domain.user.exception;
+
+import com.yanolja_final.global.exception.ApplicationException;
+import com.yanolja_final.global.exception.ErrorCode;
+
+public class EmailSendingException extends ApplicationException {
+
+    private static final ErrorCode ERROR_CODE = ErrorCode.EMAIL_SENDING_FAILURE;
+
+    public EmailSendingException() {
+
+        super(ERROR_CODE);
+    }
+
+}
```

Comment:
```suggestion
    public EmailSendingException() {
        super(ERROR_CODE);
    }
}

````

이렇게 코드 스타일 통일하는건 어떨까요?

1. 메서드 선언의 맨 위와 아래는 줄바꿈 X
2. 클래스 선언의 맨 위는 줄바꿈 O, 맨 아래는 줄바꿈 X
3. 파일의 맨 아래에 줄바꿈 삽입 (맨 아래 개행이 필요한 이유: https://seongwon.dev/Git/20220303-%ED%8C%8C%EC%9D%BC%EC%9D%98_%EB%A7%88%EC%A7%80%EB%A7%89_%EA%B0%9C%ED%96%89/)